src/settings/locationCodeValidator.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+const locationCodeValidator = {
+  validate({ code }) {
+    return !code
```

Comment:
It looks like this is an extra negation operator. I believe, that it can be removed (bodies of a conditional operator should be swapped in this case)

Reply:
For me, ternary, when the first option is longer, is more natural really. Negation is not recommended to use when variables have negation part in it (e.g. `isInvalid`). Then negation for it does not look okay `!isInvalid`. The negation itself is not an anty-pattern, in my opinion, when it does not bring the mental overhead.